app/views/posts/index.html.erb

```diff
@@ -1,48 +1,45 @@
-<div class="user_container">
-  <div class="btn_post_field">
-    <button type="button" class="all_post_btn"><%= link_to "Create post", new_user_post_path %></button>
-  </div>
-
-  <section class="user_field">
-    <article class="user_desc">
+<div class="container mt-5">
+  <section class="d-flex m-4 justify-content-between mt-4">
+    <article class="border border-secondary border-3 p-5">
+      <%= image_tag("") %>
+    </article>
+    <article class="border border-secondary border-3 p-3 w-50">
       <h2> <%= @user.name %> </h2>
-      <p class="post_counter_field"> posts status: <%= @user.posts.length %> </p>
+      <p> Number of posts: <%= @user.post_counter || 0 %> </p>
     </article>
   </section>
 
+  <div>
+    <%= link_to "Create new post", new_user_post_path, class: 'btn btn-primary d-flex justify-content-center fixed-top' %>
+  <div>
+
 <% @user.posts.each do |post| %>
-  <section class="">
-  <article class="post_field">
-    <%= link_to user_post_path(@user.id, post.id), id: "user_post #{post.id}" do %>
-      <h4> <%= post.title %> </h4>
-    <% end %>
+  <section class="border border-secondary border-3 rounded mt-3 p-2">
+    <div>
+      <%= link_to user_posts_path(post) do %>
```

Comment:
[OPTIONAL] When I click on the post link, it does not take me to the post page. It would be preferable to improve the user experience by fixing the routing link.